lib/generator.rb

```diff
@@ -14,11 +14,11 @@ def initialize(name, cases)
   end
 
   def metadata_dir
-    File.expand_path(File.join('..', '..', '..', METADATA_REPOSITORY), __FILE__)
+    File.expand_path(File.join('..', '..', '..', METADATA_REPOSITORY, 'exercises', "#{ name }"), __FILE__)
```

Comment:
No need to have redundant string interpolation here, 
```
File.expand_path(File.join('..', '..', '..', METADATA_REPOSITORY, 'exercises', name), __FILE__)
```
should work just as well.